cloudfoundry-client/src/main/java/org/cloudfoundry/client/v2/services/Services.java

```diff
@@ -24,6 +24,14 @@
 public interface Services {
 
     /**
+     * Makes the <a href="http://apidocs.cloudfoundry.org/214/services/delete_a_particular_service.html">Delete the Service</a> request
+     *
+     * @param request the Delete Service request
+     * @return the response from the Delete Service request
+     */
+    Mono<Void> delete(DeleteServiceRequest request);
```

Comment:
@antechrestos You won't know this but we've identified that despite what the documentation says, any time an API allows an `async` argument, it returns a `JobEntity`-based response.  I've [opened an issue](https://github.com/cloudfoundry/cloud_controller_ng/issues/536) and make the fix in a following commit.